mac/Keyman4MacIM/Keyman4MacIM/KMConfiguration/ha.lproj/preferences.strings

```diff
@@ -0,0 +1,38 @@
+/* Checkbox text to always show on-screen keyboard */
+"4UX-80-0Vo.title" = "Ko da yaushe a nuna madannai na allon kwamfuta";
+
+/* Checkbox text to enable verbose Console logging */
+"7J6-zy-R20.title" = "Ayi amfani da shigar verbose Console";
+
+/* Support button text  */
+"93O-x6-RLF.label" = "Tallafi";
+
+/* Download Keyboard button text */
+"CTw-kf-WNS.title" = "A sauke Madannai...";
+
+/* Keyman Configuration window title */
+"F0z-JX-Cv5.title" = "Yi amfani da shigar verbose Console";
```

Comment:
Incorrect translation -- this is the string to enable verbose console logging

Reply:
@sgschantz - I've updated this string